openedx/frontend-app-course-authoring
# fix: minor react errors in course authoring mfe [FC-0036]

## Description
This PR fixes console warnings in the Taxonomy pages and a type name fix related to TagData and TagListData (the names were swapped)

## Supporting Information
* Closes https://github.com/openedx/modular-learning/issues/171

## Testing instructions
1. Go to the taxonomy list page (http://localhost:2001/taxonomies)
2. You should only see warnings in the console related to components outside the course-authoring scope: 
  - From `StudioHeader`
```
Warning: Failed prop type: Invalid prop `buttonTitle` of type `object` supplied to `NavDropdownMenu`, expected `string`.
```
  - From `StudioFooter`
```
Warning: Failed prop type: The prop `destination` is marked as required in `ForwardRef(_c)`, but its value is `undefined`.
```
- Multiple errors from `@formatjs/intl` if you are using a locale with missing translations (you can filter them out using the string `-format` in Firefox
```
Error: [@formatjs/intl Error MISSING_TRANSLATION] Missing message: "course-authoring.taxonomy-menu.action.button.label" for locale "xx-XX", using default message as fallback. 
```
3. Navigate through the app and check if there are no new warnings shown in the console:
 - Open the Export Taxonomy Dialog
 - Open the Delete Taxonomy Dialog
 - Open the Taxonomy Detail and interact with the Tag List

___
Private-ref:
 - [FAL-3608](https://tasks.opencraft.com/browse/FAL-3608)
